### PR TITLE
log start of video with progress 0, add pause

### PR DESCRIPTION
**Pull request by lc512k**
cc @commuterjoy

### CODE REVIEW

File: server/transforms/video.js [hunk 1]
```diff
@@ -18,13 +18,17 @@ class Video extends Transform {
...
+							this.event.pluck('action') === 'playing' &&
+							this.event.pluck('context.progress') === 0;
 
-		if (!isPlayEvent) {
-			logger.info('transforms/video temporarily skipping video events other than play, playing and ended');
+		const videoEnded = this.event.pluck('category') === 'video' &&
+							this.event.pluck('action') === 'ended';
+
+		if (videoStarted || videoEnded) {
+			this.count(`pipeline.transforms.${this.graphite}.${this.event.pluck('action')}`);
```
Comment (commuterjoy) [inline server/transforms/video.js]: You could just do this,

```
this.count(this.event.pluck('action'));
```
https://github.com/Financial-Times/spoor-enrichment/blob/master/server/models/transform.js#L30

Comment (commuterjoy) [inline server/transforms/video.js]: Also, we collect the product, action, category data in graphite already :)

https://github.com/Financial-Times/spoor-enrichment/blob/master/server/filters/metrics.js

**Comment by commuterjoy** [comment]
are you going to add the ended & started flags to the response object?

+1 aside from comments

**Comment by lc512k** [comment]
> are you going to add the ended & started flags to the response object?

Just curious to what the benefit would be, if it's already the event action? Maybe only to simplify querying both action and progress for the initial play event?

**Comment by lc512k** [comment]
Merging this for now, but will add flags to the response if needed

**Pull request merged**